src/dafny/Collections/Sets.dfy

```diff
@@ -193,4 +193,35 @@ module {:options "-functionSyntax:4"} Dafny.Collections.Sets {
     LemmaSubsetSize(x, range);
   }
 
+  /** Any totally ordered set contains a unique minimal element. */
```

Comment:
The documentation mentions `minimal` elements, but the specification only says something about minimum elements.